src/action.ts

```diff
@@ -93,9 +93,13 @@ export async function run(): Promise<void> {
 	core.setOutput('version', rawVersion);
 	core.setOutput('release_id', releaseId);
 
-	if (core.getBooleanInput('create_ref', { required: false })) {
+	// originally called create_ref but was renamed to create_tag
+	if (
+		core.getBooleanInput('create_tag', { required: false }) ||
+		core.getBooleanInput('create_ref', { required: false })
```

Comment:
I thought it was worthwhile to have some backwards-compatibility here, at least for my own sake since I have this flag enabled on a dozen projects that would need to be updated manually to go to the latest and not lose the flag.

Reply:
Nice! I was thinking it's ok to break compatibility since people could / should pin to a specific version. Does the input get passed even though we removed `create_ref` as n input from the action.yml ?

Reply:
> I was thinking it's ok to break compatibility since people could / should pin to a specific version

I thought so too but then I remembered that @ab77 intentionally unpinned via https://github.com/product-os/balena-jenkins/pull/71 😱

>  Does the input get passed even though we removed create_ref as n input from the action.yml ?

It prints a warning but still seems to pass the value because the tag was created.
```
Warning: Unexpected input(s) 'create_ref', valid inputs are ['entryPoint', 'args', 'balena_token', 'fleet', 'github_token', 'environment', 'versionbot', 'finalize', 'create_tag', 'cache', 'source']
Run balena-io/deploy-to-balena-action@klutchell/rename-create-ref
  with:
    balena_token: ***
    fleet: ***
    github_token: ***
    versionbot: false
    environment: balena-cloud.com
    create_ref: true
    finalize: true
    create_tag: false
    cache: true
...
Creating refs/tags/0.0.0-1640182241828
```